docs_app/content/tutorial/2-unsubscribing.md

```diff
@@ -0,0 +1,39 @@
+# 2. Unsubscribing
+
+## TLDR:
+
+- Calls to [`subscribe`](API) return a [`Subscription`](API) object.
+- Calling [`unsubscribe`](API) on the [`Subscription`](API) object notifies the [producer](GL) to stop sending values, and triggers the [teardown](GL) of any underlying resources the [producer](GL) has registered for teardown.
+- Failure to [unsubscribe](GL) from asynchronous observables will result in unnecessary resource use, and even memory leaks.
+- Any errors that occur during teardown will be collected and rethrown as an [`UnsubscriptoinError`](API).
+
+## Overview
+
+Just as you need to know how to start getting values from your observable, you must also need to know how to tell the [producer](GL) to stop sending values, and perhaps more importantly, to [teardown](GL) and free up resources.
+
+## Subscriptions
+
+All calls to [`subscribe`](API) return a [`Subscription`](API) object. In order to end your [subscription](GL) and tell the [producer](GL) to stop sending values and [teardown](API), you must call [`unsubscribe`](API).
+
+When [`unsubscribe`](API) is called, it will synchronously trigger the [teardown](GL) of the entire [stream](GL) and all underlying [producers](GL) from [cold](GL) sources. This means from that exact moment on, no values can be [nexted](GL), and you cannot be [notified](GL) of any errors or completions.
```

Comment:
Haven't previously introduced the concept of "cold" sources. Is there another way to say that so that concept doesn't need to be introduced here?